### ISSUE TITLE
Create a site layout

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#29]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

A unified MVC layout.

### Why

* to define a uniform look & feel
* to handle global site navigation in one place

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

Resources:

* https://docs.microsoft.com/en-us/aspnet/core/mvc/views/layout?view=aspnetcore-3.1
* https://docs.kentico.com/13/developing-websites/building-website-navigation
